app/src/web.ts

```diff
@@ -0,0 +1,40 @@
+import { WebPlugin } from '@capacitor/core';
+
+import type { AppInfo, AppPlugin, AppLaunchUrl, AppState } from './definitions';
+
+export class AppWeb extends WebPlugin implements AppPlugin {
+  constructor() {
+    super({ name: 'App' });
+    if (typeof document !== 'undefined') {
+      document.addEventListener(
+        'visibilitychange',
+        this.handleVisibilityChange.bind(this),
+        false,
+      );
+    }
+  }
+
+  exitApp(): never {
+    throw new Error('Method not implemented.');
+  }
+
+  async getInfo(): Promise<AppInfo> {
+    throw new Error('Method not implemented.');
```

Comment:
Do we not know this information for Capacitor web? We could implement this in the future, maybe.

Reply:
No, we don't.
There was a proposal of copying web files to a different folder, including the capacitor.config.json, if we do that we could read the file and get the app name and app id, but version and build won't be available, unless we also copy the package.json or make the cli put some value from it on the capacitor.config.json based on the package.json version

Reply:
Yeah, that's what I was thinking. But we can do it later